buildout.cfg

```diff
@@ -53,7 +53,7 @@ zope.testrunner = 4.7.0
 [script]
 recipe = zc.recipe.egg
 dependent-scripts = true
-interpreter = python
+interpreter = py
```

Comment:
```
-interpreter = python
+interpreter = py
```

> Why's this?
> 
> (My guess: you use a virtualenv to get rid of the bootstrap.py? The change sounds OK then.)

I don't particularly want to get rid of `bootstrap.py` (I used it in fact), but I like using in-line virtualenvs, as in:

    virtualenv --no-setuptools --no-pip .

And running `bootstrap.py` after.

But as the commit message that introduced that change says, I'm changing this:

> So it doesn't conflict with an inline virtualenv.

Otherwise, running buildout causes `"Text file busy"` errors on (at least) Linux, when buildout tries to replace the Python executable that is running the buildout with a script.